### DISCUSSION TITLE
Configurable option to use album name + folder to group albums

### DISCUSSION THREAD

**Discussion by Chevrah** [navidrome/navidrome#2458] [Ideas]
This would be a great feature, to have an option that folders should be treated as one album. 

Will solve a lot of issues when there's multiple artists for one album.

**Comment by metalheim** [comment]
When there's multiple artists for one album, you should just be fine to set the "albumartist" tag (same value for all tracks).
If there is no definitive albumartist, you can set the "compilation" (TCMP) tag to 1 and navidrome will bundle the tracks as a compilation album (with "Various artists" as albumartist).

I think this is the more elegant solution rather than reling on folder structure.
Respecting and reling on folder structures can create more headaches, f.e. if you have an album with multiple artists, but its split into subfolders (think CD1 and CD2 subfolders).

IMHO (but im not deciding anything definitevly) it's more hassle than it's worth.

**Reply by Chevrah** [reply]
Right now I am tagging them with Album Artist, which causes me more headaches..

Thanks for the suggestion! 

I was just posting here in the ideas that there should be an option in the config. Not the default.
